internal/kibana/agents.go

```diff
@@ -100,70 +101,81 @@ func (c *Client) GetAgentStatusByHostname(hostname string) (string, error) {
 
 // GetAgentEvents get events of agent
 func (c *Client) GetAgentEvents(applicationName string, agentID string, packagePolicyID string, updatedAt string) error {
-	statusCode, respBody, err := c.get(fmt.Sprintf("%s/agents/%s/events", FleetAPI, agentID))
-
+	query := map[string]interface{}{
```

Comment:
I'm OK with this change, although not sure if we still want to test the kibana APIs for that. Are we sure the API disappeared instead of changed? @EricDavisX @nchaulet could you please confirm?

Reply:
Yes that APIs disappear, there is no agents events saved object on our side.

Reply:
@adam-stokes what I'm not sure is if we want to make this change in this refactor, as we pinned the kibana version until we fix the rest of the enroll scenarios 🤔

Reply:
If that API disappeared, maybe we can conclude that the scenario steps related to it could disappear too. @EricDavisX are your thoughts on this? Should we keep steps related to agent events after the removal of the API? Apart from the ES query that Adam brightly added, is there any other way to get those events?

Reply:
> @adam-stokes what I'm not sure is if we want to make this change in this refactor, as we pinned the kibana version until we fix the rest of the enroll scenarios 🤔

This change was required for the agent endpoint test suite to pass when adding the endpoint policy

Reply:
No there is no other way to get these logs otherwise than querying ES or going to the logs UI. 
I think testing that Agent correctly send logs could make sense in e2e tests but it's  probably a new scenario step.